pkgs/top-level/python-packages.nix

```diff
@@ -4756,8 +4756,6 @@ self: super: with self; {
 
   gtfs-realtime-bindings = callPackage ../development/python-modules/gtfs-realtime-bindings { };
 
-  gtimelog = callPackage ../development/python-modules/gtimelog { };
```

Comment:
This should get an entry in python-aliases.nix

Reply:
Thanks! I added `gtimelog = throw "gtimelog has been promoted to a top-level attribute"; # added 2023-10-10`. I hope that's the correct entry, wasn't entirely sure. :)